build-system/tasks/karma.conf.js

```diff
@@ -35,19 +35,26 @@ module.exports = {
     debug: true,
     fast: true,
     transform: [
-      ['babelify', {presets: ['@babel/preset-env']}],
       () =>
-        through(function (buf, enc, next) {
+        through2(function (buf, enc, next) {
           this.push(
             buf
               .toString('utf8')
-              // Set Pay environment to indicate we're in a Karma test.
-              .replace(/\$payEnvironment\$/g, 'TEST')
-              // Some tests need a valid SwG server origin.
-              .replace(/\$frontend\$/g, 'https://news.google.com')
+              /**
```

Comment:
@ChrisAntaki so this is unfortunate. Apparently theres a bunch of plugin that were reporting issues with istanbul and they made hacks to get around it with no response on the issues for the past few years

Reply:
Years? Oof